variants/frontend-react/template.rb

```diff
@@ -32,9 +32,12 @@
 gsub_file(
```

Comment:
I think this `gsub_file` should completely go away given the `prepend_to_file` you've added.

Reply:
That was to remove `var ReactRailsUJS = require("react_ujs")` from the file. 

If I remove that line (gsub) this time I have this error because the decleration is still there.

app/frontend/packs/application.ts:1:8 - error TS2440: Import declaration conflicts with local declaration of 'ReactRailsUJS'.

1 import ReactRailsUJS from 'react_ujs';
         ~~~~~~~~~~~~~

app/frontend/packs/application.ts:56:5 - error TS2440: Import declaration conflicts with local declaration of 'ReactRailsUJS'.

56 var ReactRailsUJS = require("react_ujs");
       ~~~~~~~~~~~~~

Note: the old version of this file was replacing the local declaration with an import declaration but this created issues with the new generator because it complained that all the import declarations need to be together and no gaps between them so I prepend it to the file to keep them together.

Reply:
Ah right, that makes sense 👍